R/funset.R

```diff
@@ -67,47 +68,60 @@ which_loaded <- function(x) {
 }
 
 funs_loaded <- function(x,crump=TRUE) {
-  main_loaded(x) && compiled.mrgmod(x)
+  main_loaded(x) && x@shlib$compiled
 }
 
 all_loaded <- function(x) all(which_loaded(x))  
 
-pointers <- function(x) {
+pointers <- function(x, refresh = FALSE) {
   if(!funs_loaded(x)) {
     try_load <- try(loadso(x), silent = TRUE)
     if(inherits(try_load, "try-error") || !funs_loaded(x)) {
       message(try_load)
       stop(FUNSET_ERROR__)
     }
   }
+  if(!refresh && VALIDPOINTERS(getpointers(x))) {
+    return(getpointers(x))
+  }
```

Comment:
This is new; return the list of model function pointers in the model unless we force a refresh or we find that any one of them is null